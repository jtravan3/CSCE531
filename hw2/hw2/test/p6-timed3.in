x=1
y=4
z=9
x+y+z
(#4+#4)%7919
(#5+#5)%7919
(#6+#6)%7919
(#7+#7)%7919
(#8+#8)%7919
x=(#9+#9)%7919
(#10+#10)%7919
(#11+#11)%7919
(#12+#12)%7919
(#13+#13)%7919
(#14+#14)%7919
(#15+#15)%7919
(#16+#16)%7919
(#17+#17)%7919
(#18+#18)%7919
y=(#19+#19)%7919
(#20+#20)%7919
(#21+#21)%7919
(#22+#22)%7919
(#23+#23)%7919
(#24+#24)%7919
(#25+#25)%7919
(#26+#26)%7919
(#27+#27)%7919
(#28+#28)%7919
z=(#29+#29)%7919
(#30+#30)%7919
(#31+#31)%7919
(#32+#32)%7919
(#33+#33)%7919
(#34+#34)%7919
(#35+#35)%7919
(#36+#36)%7919
(#37+#37)%7919
(#38+#38)%7919
x=(#39+#39)%7919
(#40+#40)%7919
(#41+#41)%7919
(#42+#42)%7919
(#43+#43)%7919
(#44+#44)%7919
(#45+#45)%7919
(#46+#46)%7919
(#47+#47)%7919
(#48+#48)%7919
y=(#49+#49)%7919
(#50+#50)%7919
(#51+#51)%7919
(#52+#52)%7919
(#53+#53)%7919
(#54+#54)%7919
(#55+#55)%7919
(#56+#56)%7919
(#57+#57)%7919
(#58+#58)%7919
z=(#59+#59)%7919
(#60+#60)%7919
(#61+#61)%7919
(#62+#62)%7919
(#63+#63)%7919
(#64+#64)%7919
(#65+#65)%7919
(#66+#66)%7919
(#67+#67)%7919
(#68+#68)%7919
x=(#69+#69)%7919
(#70+#70)%7919
(#71+#71)%7919
(#72+#72)%7919
(#73+#73)%7919
(#74+#74)%7919
(#75+#75)%7919
(#76+#76)%7919
(#77+#77)%7919
(#78+#78)%7919
y=(#79+#79)%7919
(#80+#80)%7919
(#81+#81)%7919
(#82+#82)%7919
(#83+#83)%7919
(#84+#84)%7919
(#85+#85)%7919
(#86+#86)%7919
(#87+#87)%7919
(#88+#88)%7919
z=(#89+#89)%7919
(#90+#90)%7919
(#91+#91)%7919
(#92+#92)%7919
(#93+#93)%7919
(#94+#94)%7919
(#95+#95)%7919
(#96+#96)%7919
(#97+#97)%7919
(#98+#98)%7919
x=(#99+#99)%7919
(#100+#100)%7919
(#101+#101)%7919
(#102+#102)%7919
(#103+#103)%7919
(#104+#104)%7919
(#105+#105)%7919
(#106+#106)%7919
(#107+#107)%7919
(#108+#108)%7919
a=(#109+#109)%7919
(#110+#110)%7919
(#111+#111)%7919
(#112+#112)%7919
(#113+#113)%7919
(#114+#114+a)%7919
(#115+#115)%7919
(#116+#116)%7919
(#117+#117)%7919
(#118+#118)%7919
y=(#119+#119)%7919
(#120+#120)%7919
(#121+#121)%7919
(#122+#122)%7919
b=(#123+#123)%7919
(#124+#124)%7919
(#125+#125*b)%7919
(#126+#126)%7919
(#127+#127)%7919
(#128+#128)%7919
z=(#129+#129)%7919
(#130+#130)%7919
(#131+#131)%7919
c=(#132+#132)%7919
(#133+#133)%7919
(#134*#17+#134)%7919
(#135+#135)%7919
d=(#136*c+#136)%7919
(#137+#137)%7919
(#138+#138)%7919
x=(#139+#139)%7919
(#140+#140)%7919
(#141+#141)%7919
(#142+#142)%7919
(#143+#143)%7919
(#144+#144)%7919
d=(#145+#145+d)%7919
(#146+#146)%7919
(#147+#147)%7919
(#148+#148)%7919
y=(#149+#149)%7919
(#150+#150)%7919
(#151+#151)%7919
(#152+#152)%7919
(#153+#153)%7919
(#154+#154)%7919
(#155+#155)%7919
(#156+#156)%7919
(#157+#157)%7919
(#158+#158)%7919
z=(#159+#159)%7919
(#160+#160)%7919
(#161+#161)%7919
(#162+#162)%7919
(#163+#163)%7919
(#164+#164)%7919
(#165+#165)%7919
(#166+#166)%7919
(#167+#167)%7919
(#168+#168)%7919
x=(#169+#169)%7919
(#170+#170)%7919
(#171+#171)%7919
(#172+#172)%7919
(#173+#173)%7919
(#174+#174)%7919
(#175+#175)%7919
(#176+#176)%7919
(#177+#177)%7919
(#178+#178)%7919
y=(#179+#179)%7919
(#180+#180)%7919
(#181+#181)%7919
(#182+#182)%7919
(#183+#183)%7919
(#184+#184)%7919
(#185+#185)%7919
(#186+#186)%7919
(#187+#187)%7919
(#188+#188)%7919
z=(#189+#189)%7919
(#190+#190)%7919
(#191+#191)%7919
(#192+#192)%7919
(#193+#193)%7919
(#194+#194*d)%7919
(#195+#195)%7919
(#196+#196)%7919
(#197+#197)%7919
(#198+#198)%7919
x=(#199+#199)%7919
(#200+#200)%7919
(#201+#201)%7919
(#202+#202)%7919
(#203+#203)%7919
(#204+#204)%7919
(#205+#205)%7919
(#206+#206)%7919
(#207+#207)%7919
(#208+#208)%7919
(#209+#209)%7919
(#210+#210)%7919
(#211+#211)%7919
(#212+#212)%7919
(#213+#213)%7919
(#214+#214)%7919
(#215+#215)%7919
(#216+#216)%7919
(#217+#217)%7919
(#218+#218)%7919
y=(#219+#219)%7919
(#220+#220)%7919
(#221+#221)%7919
(#222+#222)%7919
(#223+#223)%7919
(#224+#224)%7919
(#225+#225)%7919
(#226+#226)%7919
(#227+#227)%7919
(#228+#228)%7919
z=(#229+#229)%7919
(#230+#230)%7919
(#231+#231)%7919
(#232+#232)%7919
(#233+#233)%7919
(#234+#234)%7919
(#235+#235)%7919
(#236+#236)%7919
(#237+#237)%7919
(#238+#238)%7919
x=(#239+#239)%7919
(#240+#240)%7919
(#241+#241)%7919
(#242+#242)%7919
(#243+#243)%7919
(#244+#244)%7919
(#245+#245)%7919
(#246+#246)%7919
(#247+#247)%7919
(#248+#248)%7919
y=(#249+#249)%7919
(#250+#250)%7919
(#251+#251)%7919
(#252+#252)%7919
(#253+#253)%7919
(#254+#254)%7919
(#255+#255)%7919
(#256+#256)%7919
(#257+#257)%7919
(#258+#258)%7919
z=(#259+#259)%7919
(#260+#260)%7919
(#261+#261)%7919
(#262+#262)%7919
(#263+#263)%7919
(#264+#264)%7919
(#265+#265)%7919
(#266+#266)%7919
(#267+#267)%7919
(#268+#268)%7919
x=(#269+#269)%7919
(#270+#270)%7919
(#271+#271)%7919
(#272+#272)%7919
(#273+#273)%7919
(#274+#274)%7919
(#275+#275)%7919
(#276+#276)%7919
(#277+#277)%7919
(#278+#278)%7919
y=(#279+#279)%7919
(#280+#280)%7919
(#281+#281)%7919
(#282+#282)%7919
(#283+#283)%7919
(#284+#284)%7919
(#285+#285)%7919
(#286+#286)%7919
(#287+#287)%7919
(#288+#288)%7919
z=(#289+#289)%7919
(#290+#290)%7919
(#291+#291)%7919
(#292+#292)%7919
(#293+#293)%7919
(#294+#294)%7919
(#295+#295)%7919
(#296+#296)%7919
(#297+#297)%7919
(#298+#298)%7919
x=(#299+#299)%7919
(#300+#300)%7919
(#301+#301)%7919
