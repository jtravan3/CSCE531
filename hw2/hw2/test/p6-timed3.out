Welcome to the Simple Expression Evaluator.
Enter one expression per line, end with ^D

   1: 1
   2: 4
   3: 9
   4: 14
   5: 28
   6: 56
   7: 112
   8: 224
   9: 448
  10: 896
  11: 5486
  12: 3053
  13: 6106
  14: 4293
  15: 667
  16: 1334
  17: 2668
  18: 5336
  19: 2753
  20: 5506
  21: 1664
  22: 3328
  23: 6656
  24: 5393
  25: 2867
  26: 5734
  27: 3549
  28: 7098
  29: 6277
  30: 4635
  31: 4653
  32: 1387
  33: 2774
  34: 5548
  35: 3177
  36: 6354
  37: 4789
  38: 1659
  39: 3318
  40: 6636
  41: 6237
  42: 4555
  43: 1191
  44: 2382
  45: 4764
  46: 1609
  47: 3218
  48: 6436
  49: 4953
  50: 1987
  51: 3148
  52: 6296
  53: 4673
  54: 1427
  55: 2854
  56: 5708
  57: 3497
  58: 6994
  59: 6069
  60: 4219
  61: 1413
  62: 2826
  63: 5652
  64: 3385
  65: 6770
  66: 5621
  67: 3323
  68: 6646
  69: 5373
  70: 2827
  71: 1577
  72: 3154
  73: 6308
  74: 4697
  75: 1475
  76: 2950
  77: 5900
  78: 3881
  79: 7762
  80: 7605
  81: 2688
  82: 5376
  83: 2833
  84: 5666
  85: 3413
  86: 6826
  87: 5733
  88: 3547
  89: 7094
  90: 6269
  91: 2993
  92: 5986
  93: 4053
  94: 187
  95: 374
  96: 748
  97: 1496
  98: 2992
  99: 5984
 100: 4049
 101: 3378
 102: 6756
 103: 5593
 104: 3267
 105: 6534
 106: 5149
 107: 2379
 108: 4758
 109: 1597
 110: 3194
 111: 6388
 112: 4857
 113: 1795
 114: 3590
 115: 2455
 116: 4910
 117: 1901
 118: 3802
 119: 7604
 120: 7289
 121: 7735
 122: 7551
 123: 7183
 124: 6447
 125: 4975
 126: 6850
 127: 5781
 128: 3643
 129: 7286
 130: 6653
 131: 7124
 132: 6329
 133: 4739
 134: 1559
 135: 2183
 136: 4366
 137: 2493
 138: 4986
 139: 2053
 140: 4106
 141: 3298
 142: 6596
 143: 5273
 144: 2627
 145: 5254
 146: 5082
 147: 7423
 148: 6927
 149: 5935
 150: 3951
 151: 3579
 152: 7158
 153: 6397
 154: 4875
 155: 1831
 156: 3662
 157: 7324
 158: 6729
 159: 5539
 160: 3159
 161: 5163
 162: 2407
 163: 4814
 164: 1709
 165: 3418
 166: 6836
 167: 5753
 168: 3587
 169: 7174
 170: 6429
 171: 447
 172: 894
 173: 1788
 174: 3576
 175: 7152
 176: 6385
 177: 4851
 178: 1783
 179: 3566
 180: 7132
 181: 1756
 182: 3512
 183: 7024
 184: 6129
 185: 4339
 186: 759
 187: 1518
 188: 3036
 189: 6072
 190: 4225
 191: 4910
 192: 1901
 193: 3802
 194: 7604
 195: 6412
 196: 4905
 197: 1891
 198: 3782
 199: 7564
 200: 7209
 201: 4655
 202: 1391
 203: 2782
 204: 5564
 205: 3209
 206: 6418
 207: 4917
 208: 1915
 209: 3830
 210: 7660
 211: 7401
 212: 6883
 213: 5847
 214: 3775
 215: 7550
 216: 7181
 217: 6443
 218: 4967
 219: 2015
 220: 4030
 221: 2637
 222: 5274
 223: 2629
 224: 5258
 225: 2597
 226: 5194
 227: 2469
 228: 4938
 229: 1957
 230: 3914
 231: 2700
 232: 5400
 233: 2881
 234: 5762
 235: 3605
 236: 7210
 237: 6501
 238: 5083
 239: 2247
 240: 4494
 241: 5273
 242: 2627
 243: 5254
 244: 2589
 245: 5178
 246: 2437
 247: 4874
 248: 1829
 249: 3658
 250: 7316
 251: 5955
 252: 3991
 253: 63
 254: 126
 255: 252
 256: 504
 257: 1008
 258: 2016
 259: 4032
 260: 145
 261: 6125
 262: 4331
 263: 743
 264: 1486
 265: 2972
 266: 5944
 267: 3969
 268: 19
 269: 38
 270: 76
 271: 5423
 272: 2927
 273: 5854
 274: 3789
 275: 7578
 276: 7237
 277: 6555
 278: 5191
 279: 2463
 280: 4926
 281: 1526
 282: 3052
 283: 6104
 284: 4289
 285: 659
 286: 1318
 287: 2636
 288: 5272
 289: 2625
 290: 5250
 291: 6461
 292: 5003
 293: 2087
 294: 4174
 295: 429
 296: 858
 297: 1716
 298: 3432
 299: 6864
 300: 5809
 301: 3443
 302: 6886
 303: 
